configdisk/playbooks/50-firewall.yml

```diff
@@ -0,0 +1,18 @@
+---
+- name: 50-firewall
+  hosts: localhost
+  vars:
+    firewall:
+      - port:
+          - "22/tcp"
```

Comment:
for ssh, it is probably better to use service and not port (if the role can do it).